### DISCUSSION TITLE
`<details>` in Safari doesn't render the arrow

### DISCUSSION THREAD

**Discussion by dorianmariecom** [community/community#16431] [Other Feature Feedback, Questions, & Ideas] [answered]
https://gist.github.com/ericclemmons/b146fe5da72ca1f706b2ef72a20ac39d

In Chrome:

<img width="368" alt="Screen Shot 2022-05-12 at 09 09 49" src="https://user-images.githubusercontent.com/58794487/168012394-a64b19ba-1533-4bb5-abd3-406b94ba6da5.png">

In Safari:

<img width="339" alt="Screen Shot 2022-05-12 at 09 10 16" src="https://user-images.githubusercontent.com/58794487/168012484-b42141a2-0878-433c-bcc4-1d312102f607.png">

Might be a Safari bug, not sure

Latest macOS (12.2.1)

**Comment by beerisgood** [comment]
I can confirm this with Safari under macOS 12.3.1

These arrow's works: https://www.w3schools.com/howto/howto_css_arrows.asp

**Reply by ghost** [reply]
Also confirmed with Safari 15.5 on MacOS 12.4.

I found that if I choose _Disable Styles_ (under the _Develop_) menu, then the arrows show up. So I guess it’s a CSS issue?

**Comment by kohlmannj** [comment]
Also ran into this! Big fan of the `<details>` element on GitHub, and Safari is my primary browser. Thus I'd love to see a fix soon. No rush, though, thanks 🙏🏼

**Comment by kohlmannj** [comment, marked as answer]
Looks like this is fixed! Tested in macOS Safari 15.5 (17613.2.7.1.8) on Monterey 12.4 (21F79):

<details><summary>Thanks a lot!</summary>
<br>
Really appreciate the fix 🥳
</details>

<img width="640" alt="Screen Shot 2022-06-09 at 12 46 45 PM" src="https://user-images.githubusercontent.com/191304/172900811-5d712190-1024-4763-9cf6-bcc8252406d2.png">

**Reply by beerisgood** [reply]
Fixed also on iOS 15.5 👍
